CapitalForecasting/Modules/processing_and_eda.R

```diff
@@ -69,32 +48,58 @@ work_cap %>%
   components() %>%
   autoplot()
 
+# more specific seasonality
+work_cap %>%
+  gg_tsdisplay(difference(working_capital),
+               plot_type = 'partial',
+               lag = 36) + labs(title = 'seasonal')
+
+# differencing
+ggplot(work_cap, aes(date, difference(working_capital))) +
+  geom_line()
+
```

Comment:
Ideally we should commit these plots too either as .png files or within an R Markdown as part of the PDF output.